src/differentials/one.jl

```diff
@@ -3,7 +3,7 @@
 The Differential which is the multiplicative identity.
 Basically, this represents `1`.
 """
-struct One <: AbstractDifferential end
+struct OneTangent <: AbstractTangent end
```

Comment:
Do we want this to be consistent with `ZeroTangent`? It seems to be used very rarely anyway.

Reply:
I think we want to delete this. https://github.com/JuliaDiff/ChainRulesCore.jl/issues/354
Can we try deprecating it in favour of `true` and see if that breaks anything on our reverse dependency tests?

`@deprecate One() true`